CondFormats/Serialization/interface/eos/portable_iarchive.hpp

```diff
@@ -371,13 +371,13 @@ namespace eos {
     /** 
 		 * \brief Load floating point types.
 		 * 
-		 * We simply rely on fp_traits to set the bit pattern from the (unsigned)
+		 * We simply rely on fp_traits_non_native to set the bit pattern from the (unsigned)
 		 * integral type that was stored in the stream. Francois Mauger provided
 		 * standardized behaviour for special values like inf and NaN, that need to
 		 * be serialized in his application.
 		 *
 		 * \note by Johan Rade (author of the floating point utilities library):
-		 * Be warned that the math::detail::fp_traits<T>::type::get_bits() function 
+		 * Be warned that the math::detail::fp_traits_non_native<T,U>::get_bits() function

```

Comment:
hi @guitargeek is this comment still true with `fp_traits_non_native` ?

Reply:
I think so, yes, because before `fp_traits<T>` was resolving to `fp_traits_non_native::type` anyway, so is must come from there (`fp_traits_non_native` is the only `fp_traits` implementation that has a `get_bits` member, `fp_traits_native` doesn't have it)